packages/protocol/contracts/governance/Election.sol

```diff
@@ -929,7 +933,8 @@ contract Election is
 
   /**
    * @notice Returns lists of all validator groups and the number of votes they've received.
-   * @return Lists of all validator groups and the number of votes they've received.
+   * @return lists Lists of all validator groups
```

Comment:
No need for first *lists*.
*List* should be singular. Actually noticing this is also a typo above, should be *list* in the `@notice` as well.